ilc/spec/hashFn.spec.js

```diff
@@ -0,0 +1,22 @@
+import chai from 'chai';
+import hashFn from '../common/hashFn';
+
+describe('hashFn', () => {
```

Comment:
I would say that these tests a little bit useless :) Let's discuss tomorrow

Reply:
Removed tests